components/nav-modal/modal.css

```diff
@@ -0,0 +1,76 @@
+:root {
+    --grey1: grey;
+    --grey2: rgb(100, 97, 97);
+    --grey: rgb(160, 156, 156);
+    --grey-dark: rgb(98, 97, 97);
+    --black: black;
+    --white: white;
+    --blue-dark: rgb(54, 89, 246);
+    --blue: rgb(26, 60, 212);
+}
+
+.modal-container {
+    width: 27rem;
+    font-size: 25px;
+    border: 1px solid var(--grey);
+}
+
+.modals {
+    padding: 1rem;
+    height: 5rem;
+}
+
+.modal-row1 {
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid var(--grey);
+}
+
+.modal-row3 {
+    display: flex;
+    justify-content: right;
+    border-top: 1px solid var(--grey);
+}
+
+.modal-close {
+    font-size: 34px;
+    color: var(--grey-dark);
+    background-color: var(--white);
+    border: none;
+
+}
+
+.modal-close:hover {
+    color: var(--black);
+}
+
+.modal-row2 {
+    font-size: 16px;
+}
+
+.modal-row3 button {
+    font-size: 18px;
```

Comment:
Change px to rem for good responsiveness.

Reply:
I have made the change,thanks for the review